### ISSUE TITLE
NTR: Ultrasound: adult worms visualized (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9030]
New term request Ultrasound: adult worms visualized (nadea3_190502015432): Visible in intestine, liver, biliary tract, lymphatic system; lymph nodes, scrotum

**Comment by pnrobinson** [comment]
@MickeySegal
this covers a lot of ground.
I am thinking we could start with a term as follows
label: Intestinal worm
def: Presence of one or more adult worms in the intestine as demonstrated by sonography or passage of a worm in the stool. On sonography, worms may appear as linear, echogenic structures within the intestines.

thoughts?

**Comment by MickeySegal** [comment]
This works for https://www.ncbi.nlm.nih.gov/books/NBK560525/.  

It doesn't work for https://www.ncbi.nlm.nih.gov/books/NBK556012/.  So I need an additional HPO code:

**Ultrasound: adult worms visualized in scrotum** in male (the article says "Scrotal ultrasound - may reveal movements of adult worms in otherwise asymptomatic males. The "filarial dance sign" is a pattern of worm movement that can be detected in the lymphatic vessels of infected men."

The article also notes that worms "can be identified to the genus level using size, body wall composition, thickness, and morphology of the cuticle. The presence of small filarial worms in lymph nodes is pathognomonic for either Wuchereria or Brugia. Adult worms are typically found in lymph nodes in the groin or neck, whereas microfilariae are typically found in the peripheral blood".  But this is not mentioned under Evaluation so it is not clear if such testing is recommended.

**Comment by pnrobinson** [comment]
@MickeySegal Adding new term: 
- ID: HP:6001038
- label: Intestinal worm
- parents: [Positive infectious agent test (HP:0430030)](https://hpo.jax.org/app/browse/term/HP:0430030)
- pmid: PMID:27551336
- def: Presence of one or more adult worms in the intestine as demonstrated by sonography or passage of a worm in the stool. On sonography, worms may appear as linear, echogenic structures within the intestines.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9030
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001039
- label: Scrotal worm
- parents: [Positive infectious agent test (HP:0430030)](https://hpo.jax.org/app/browse/term/HP:0430030)
- pmid: PMID:12636874
- def: Demonstration of one or more worms in the scrotum. This finding can be appreciated by sonography, where echogenic, linear structures could be seen moving within the scrotum.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9030
- synonyms: Filaria dance sign(NONE)
